### DISCUSSION TITLE
F1 Live Timing feature request - Circular Timing Map

### DISCUSSION THREAD

**Discussion by mbryant2** [f1multiviewer/issue-tracker#538] [Feature requests]
### Overview:

Introduce a new feature in the MultiViewer app as part of live timing that visualizes the relative gaps between drivers using a circular track representation. This visualization leverages existing telemetry data (such as mini sectors or track map data) to provide users with an intuitive and strategic overview of driver positions and gaps.

### Background:

During the F1TV Brazil coverage, particularly in a rain-out session, there was a walk-through of the McLaren garage. They quickly showed the screens that VIPs in the garage watch, which shows a circular representation of the track to monitor real-time gaps between drivers. This method normalizes the track into a circle, allowing for a clear and immediate understanding of where each driver is relative to others, which is crucial for strategic decisions like pit stops.

#### Circular Track Representation:
Normalization of Track Layout: Convert the actual track layout into a simplified circular diagram, maintaining the correct sequence of sectors.
#### Driver Markers: 
Each driver is represented by a marker (e.g., car number or Initials) placed around the circle according to their real-time position on the track.
#### Relative Gap Visualization:
Accurate Gap Distances: The spacing between driver markers corresponds proportionally to the actual gaps on the track.
Dynamic Updates: Real-time telemetry ensures that driver positions and gaps are updated live as the race progresses.
#### Sector and Mini-Sector Markings:
Sector Division: The circle is divided into the official track sectors (Sector 1, Sector 2, Sector 3), with optional mini-sector markings for more granular detail.
Visual Cues: Simple Lines or shading can denote different sectors to enhance visual clarity.

### Interactive Features (Optional Enhancements):
Driver Selection: Users can click on a driver marker to highlight them, view detailed statistics, or compare them directly with another driver.
Event Indicators: Flags or icons to indicate events like pit stops, yellow flags, or incidents.

**Comment by LapsTimeOFF** [comment]
Duplicate of #467
